text/metrics/0113-exemplars.md

```diff
@@ -0,0 +1,104 @@
+# Integrate Exemplars with Metrics
+
+This OTEP adds exemplar support to aggregations defined in the Metrics SDK.
+
+## Definition
+
+Exemplars are example data points for aggregated data. They provide specific context to otherwise general aggregations. For histogram-type metrics, exemplars are points associated with each bucket in the histogram giving an example of what was aggregated into the bucket. Exemplars are augmented beyond just measurements with references to the sampled trace where the measurement was recorded and labels that were attached to the measurement.
+
+## Motivation
+
+Defining exemplar behaviour for aggregations allows OpenTelemetry to support exemplars in Google Cloud Monitoring.
+
+Exemplars provide a link between metrics and traces. Consider a user using a Histogram aggregation to track response latencies over time for a high QPS server. The histogram is composed of buckets based on the speed of the request, for example, "there were 55 requests that took 400-500 milliseconds". The user wants to troubleshoot slow requests, so they would need to find a trace where the latency was high. With exemplars, the user is able to get an exemplar trace from a high latency bucket, an exemplar trace from a low latency bucket, and compare them to figure out the reason for the high latency.
+
+Exemplars are meaningful for all aggregations where relevant traces can provide more context to the aggregation, as well as when exemplars can display specific information not otherwise shown in the aggregation (for example, the full set of labels where they otherwise might be aggregated away).
+
+## Internal details
+
+An exemplar is defined as:
+
+```
+message RawValue {
+  // Numerical value of the measurement that was recorded. Only one of these two fields is
+  // used for the data, depending on its type
+  double double_value = 0;
+  int64 int64_value = 1;
```

Comment:
I think it would have to be the same type as the descriptor of the enclosing message.

Reply:
Hmm.. in case of Histogram/Summary we don't have that. Either add it or we need to do something else. But happy to discuss this in the proto change

Reply:
Fair point, I think having an `InputType` or similar on the metric descriptor would make the most sense. It could also be different depending on if people approve of [this](https://github.com/open-telemetry/oteps/pull/113#issuecomment-642992182) comment or not. Happy to talk about this in proto as well

Reply:
Continue here: https://github.com/open-telemetry/opentelemetry-proto/pull/159